wandb/sdk/launch/runner/gcp_vertex.py

```diff
@@ -103,12 +103,12 @@ def run(self, launch_project: LaunchProject) -> Optional[AbstractRun]:
         gcp_staging_bucket = resource_args.get("gcp_staging_bucket")
         if not gcp_staging_bucket:
             raise LaunchError(
-                "Vertex requires a staging bucket for training and dependency packages in the same region as compute. You can specify a bucket with --resource-arg gcp_staging_bucket=<bucket>."
+                "Vertex requires a staging bucket for training and dependency packages in the same region as compute. Specify a bucket under key gcp_staging_bucket."
```

Comment:
We should put up a PR for CLI for these changes as well. (Not today)

Reply:
wait what do you mean

Reply:
These are good fixes to have in master. There's a few PRs in this push that we'll end up wanting in master as well. These changes included